<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Page0_WelcomePage">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>20</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Bun venit pe forumul pentru POO</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>160</y>
          <width>201</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="Home_LogIn_Button">
           <property name="text">
            <string>Log In</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Home_SignUp_Button">
           <property name="text">
            <string>Sign Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Home_Guest_Button">
           <property name="text">
            <string>Log in as Guest</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Page1_LogIn">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>501</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Username:  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Password:  </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="LineEdit_Username"/>
             </item>
             <item>
              <widget class="QLineEdit" name="LineEdit_Password"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="LogInPage_LogIn_Button">
             <property name="text">
              <string>Log In</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LogInPage_Cancel_Button">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Page2_SignUp"/>
      <widget class="QWidget" name="Page3_Home">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>90</y>
          <width>271</width>
          <height>391</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="OtherTopics">
           <property name="text">
            <string>Other Topics</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Topic2">
           <property name="text">
            <string>Topic 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Topic4">
           <property name="text">
            <string>Topic 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Topic3">
           <property name="text">
            <string>Topic 4</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
